tcms/testruns/templates/testruns/mutable.html

```diff
@@ -74,8 +74,6 @@
         </div>
 
         {% if test_cases %}
-        {# Working around for the bug https://bugzilla.redhat.com/show_bug.cgi?id=715499 #}
-        <input type="hidden" name="POSTING_TO_CREATE" value="YES" />
```

Comment:
I want to pass all confirmed selected testcases as query params but obliviously there was a bug about that. Any suggestions?

Reply:
This bug is "Can not create new test run with more than 700 cases" with the underlying root case:

"Request-URI Too Large,The requested URL's length exceeds the capacity limit for this server."

Which apparently is controlled by the [LimitRequestLine](http://httpd.apache.org/docs/2.4/mod/core.html#limitrequestline) httpd variable with a default of 8190 bytes.

Assuming 10 chars on average (`case=xxxx&`) that is 800 cases. You can try:
- using different query names, e.g. `p=` for plan, `c=` for case
- bumping this to 100KiB (should allow > 10K cases)
^^^ will need a test but that is way too extreme.  I would say 1000 to 2000 cases could be acceptable as the limit (e.g. 10KiB length of the URI) and then add additional cases afterwards.

Another option is keeping the selected cases in a cookie and reading that from the GET request which TBH seems better. 

FTR we've got `LimitRequestBody 10485760` which is 10 MiB for POST requests.